projects/components/src/link/link.component.ts

```diff
@@ -18,6 +18,7 @@ import { EMPTY, Observable } from 'rxjs';
       [queryParamsHandling]="navData?.extras?.queryParamsHandling"
       [skipLocationChange]="navData?.extras?.skipLocationChange"
       [replaceUrl]="navData?.extras?.replaceUrl"
+      htTrack="{{ navData && navData.path ? navData.path : '' }}"
```

Comment:
an example, (which we also want to keep) :)

Reply:
but not in this PR!